src/Basics/Extra.elm

```diff
@@ -54,7 +54,7 @@ smaller than that, and behaviour becomes mathematically unsound.
 -}
 minSafeInteger : number
 minSafeInteger =
-    -9007199254740991
+    - maxSafeInteger
 
```

Comment:
Hi @zwilias, turns out this line was a syntax error. Had to make [this change](c832fc5012800f3f786bb48682abd0e42c081031) before publishing a new version of the package. Just telling you in case you think that line should have been accepted by the compiler as it was in your commit. In that case, maybe open a syntax-related ticket in the compiler repository?